storybook/src/elements/nuxeo-data-list/nuxeo-data-list.stories.js

```diff
@@ -0,0 +1,86 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { analyse } from '../../../.storybook/analysis';
+import { LIST, LIST_EMPTY } from '../../data/lists.data.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-data-list/nuxeo-data-list.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-document-thumbnail/nuxeo-document-thumbnail.js';
+
+const docs = analyse('nuxeo-data-list').notes;
+const stories = storiesOf('UI/nuxeo-data-list', module);
+const _computedClass = (isSelected) => {
```

Comment:
I don't see we using the `isSelected` in any of the calls.

Reply:
I will clean it up.